index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Trial Project</title>
+    <style>
+      * {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+      }
+      .bt {
+        background-color: #04aa6d;
+        font-size: large;
+        width: 100%;
+        padding: 12px 20px;
+        font-weight: bold;
+        border: none;
+        color: aliceblue;
+        border-radius: 3px;
+      }
+      header{
+        width: 100%;
+        height:60px;
+        padding: 7px;
+        background-color: rgb(7, 7, 7);
+        display: flex;
+        justify-content:space-between;
+        align-items: center;
+        color:blanchedalmond;
+      }      
+    body {
+    background-image: url("bg.jpg");
+    background-color: #9b9ba7;
+    }
+    .logo,.head-navigation{
+      height:50px;
+      display: flex;
+      flex-wrap: wrap;
+    }
+    .logo{
+      max-width: 250px;
+       }
+    
+
+    .logo-fig{
+      width: 50px;
+      height: 50px;
+      border:1px solid gray;
+    }
+    .logo-text{
+      width:160px;
+      height:20px;
+      border: 1px solid red;
+    }
+   
+      .b1 {
+        font-size: 36px;
+        font-weight: bold;
+        line-height: 1.2em;
+        text-align: center;
+        color: rgb(233, 233, 233);
+      }
+      .b2 {
+        text-align: center;
+        color: rgb(233, 233, 233);
+        font-style: italic;
+      }
+      .b3 {
+        font-size: 18px;
+        font-weight: bold;
+      }
+
+      .container {
+        max-width: 650px;
+        margin: 0 auto;
+        padding: 30px 30px;
+        border: 1px solid rgb(23, 23, 43);
+        background-color: rgba(5, 5, 31, 0.7);
+        color: white;
+        border-radius: 5px;
+      }
+      #name,#email,#number,#dropdown{
```

Comment:
```suggestion
        border-radius: 5px;
      }
      
      #name,
      #email,
      #number,
      #dropdown{
```
I would like to suggest you put an empty line before the beginning of a new code and a space before opening braces. It will help in avoiding linter errors. :expressionless: